html/rendering/the-details-element/summary-display-list-item-002-ref.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Test: summary with 'display: list-item' and flex children</title>
+<link rel="author" title="Cameron McCormack" href="mailto:">
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com">
+<link rel="match" href="summary-display-list-item-002-ref.html">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements">
```

Comment:
I'd drop this and only link the spec in the test.

Reply:
done